codegen/smithy-kotlin-codegen/src/main/kotlin/software/amazon/smithy/kotlin/codegen/rendering/protocol/HttpProtocolClientGenerator.kt

```diff
@@ -338,21 +337,14 @@ open class HttpProtocolClientGenerator(
 
     /**
      * Render optionally installing Md5ChecksumMiddleware.
-     * The Md5 middleware will only be installed if the operation requires a checksum and the user has not opted-in to flexible checksums.
+     * The Md5 middleware will only be installed if the operation requires a checksum.
      */
```

Comment:
question: Why is this installed for every operation now? Doesn't CRC32 get used by default?